lib/rspec/json_expectations/matchers.rb

```diff
@@ -1,3 +1,5 @@
+require "forwardable"
```

Comment:
This should address #32 reviewing the existing code it looks like this project has all the `require`s in a single file, lib/rspec/json_expectations.rb. @waterlink I recommend merging this code, or updating `json_expectations.rb` to also require forwardable.